client/my-sites/domains/domain-management/dns/restore-default-a-records-dialog.tsx

```diff
@@ -26,17 +36,18 @@ function RestoreDefaultARecordsDialog( { onClose, visible, defaultRecords } ) {
 		},
 	];
 
-	const message = defaultRecords
-		? sprintf(
```

Comment:
I couldn't find a path where this untranslated text was used, defaultRecords was always passed in as null, unless I'm missing something. Removing it seemed the best path forward, it's not translated anyway

Reply:
I think that parameter was used in the past, but not anymore